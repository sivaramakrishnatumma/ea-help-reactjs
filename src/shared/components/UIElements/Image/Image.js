import React from "react";

const Image = props => {
  return (
    <img
      src={props.src}
      alt={props.name}
      className={"img-fluid " + props.className}
    />
  );
};
export default Image;
