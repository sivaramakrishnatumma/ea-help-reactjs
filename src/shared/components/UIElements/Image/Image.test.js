import React from "react";
import Image from "./Image";
import { mount } from "enzyme";

describe("Image component test case", () => {
  it("should test Image component with default props", () => {
    const props = {
      className: "",
      alt: "submit",
      src: "abc.png"
    };
    const wrapper = mount(<Image {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
