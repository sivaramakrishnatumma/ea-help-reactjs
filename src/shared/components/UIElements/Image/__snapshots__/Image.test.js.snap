// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component test case should test Image component with default props 1`] = `
<Image
  alt="submit"
  className=""
  src="abc.png"
>
  <img
    className="img-fluid "
    src="abc.png"
  />
</Image>
`;
